--- Image_Text-0.6.0/usr/share/pear/Image/Text.php~	2010-11-16 16:49:01.000000000 +0200
+++ Image_Text-0.6.0/usr/share/pear/Image/Text.php	2010-11-16 16:50:25.906078543 +0200
@@ -230,7 +230,7 @@
             'enable_alpha'      => false,
 
             // font settings
-            'font_path'         => "./",
+            'font_path'         => "/usr/share/fonts/TTF/",
             'font_file'         => null,
             'font_size'         => 2,
             'line_spacing'      => 0.5,
